src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -28,30 +28,37 @@ static ucs_config_field_t uct_cuda_ipc_iface_config_table[] = {
 
     {"MAX_POLL", "16",
      "Max number of event completions to pick during cuda events polling",
-      ucs_offsetof(uct_cuda_ipc_iface_config_t, max_poll), UCS_CONFIG_TYPE_UINT},
+      ucs_offsetof(uct_cuda_ipc_iface_config_t, self.max_poll), UCS_CONFIG_TYPE_UINT},
 
     {"MAX_STREAMS", "16",
      "Max number of CUDA streams to make concurrent progress on",
-      ucs_offsetof(uct_cuda_ipc_iface_config_t, max_streams), UCS_CONFIG_TYPE_UINT},
+      ucs_offsetof(uct_cuda_ipc_iface_config_t, self.max_streams), UCS_CONFIG_TYPE_UINT},
 
     {"CACHE", "y",
      "Enable remote endpoint IPC memhandle mapping cache",
-     ucs_offsetof(uct_cuda_ipc_iface_config_t, enable_cache),
+     ucs_offsetof(uct_cuda_ipc_iface_config_t, self.enable_cache),
      UCS_CONFIG_TYPE_BOOL},
 
     {"ENABLE_GET_ZCOPY", "auto",
      "Enable get operations except for platforms known to have slower performance",
-     ucs_offsetof(uct_cuda_ipc_iface_config_t, enable_get_zcopy),
+     ucs_offsetof(uct_cuda_ipc_iface_config_t, self.enable_get_zcopy),
      UCS_CONFIG_TYPE_ON_OFF_AUTO},
 
     {"MAX_EVENTS", "inf",
      "Max number of cuda events. -1 is infinite",
-     ucs_offsetof(uct_cuda_ipc_iface_config_t, max_cuda_ipc_events), UCS_CONFIG_TYPE_UINT},
+     ucs_offsetof(uct_cuda_ipc_iface_config_t, self.max_cuda_ipc_events), UCS_CONFIG_TYPE_UINT},
 
     {"BW", "auto",
      "Effective p2p memory bandwidth",
-     ucs_offsetof(uct_cuda_ipc_iface_config_t, bandwidth), UCS_CONFIG_TYPE_BW},
+     ucs_offsetof(uct_cuda_ipc_iface_config_t, self.bandwidth), UCS_CONFIG_TYPE_BW},
 
+    {"LAT", "1.8us",
+     "Estimated latency",
+     ucs_offsetof(uct_cuda_ipc_iface_config_t, self.latency), UCS_CONFIG_TYPE_TIME},
+
+    {"OVERHEAD", "4.0us",
+     "Estimated CPU overhead for operation of transferring GPU memory",
```

Comment:
maybe
`Estimated CPU overhead for transferring GPU memory`

Reply:
Done